.github/workflows/release.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         run: cargo install wasm-pack
 
       - name: Build
-        run: wasm-pack build --target web --out-dir dist --release
+        run: wasm-pack build --target web --out-dir dist --release --scope dfinity
```

Comment:
it isn't @dfinity ?

Reply:
No, the `@` is added by `wasm-pack`. I.e. the resulting `name` in the `package.json` is now `@dfinity/standalone-sig-verifier-web`.